cpp/src/parquet/encoding.cc

```diff
@@ -4055,4 +4055,57 @@ std::unique_ptr<Decoder> MakeDictDecoder(Type::type type_num,
 }
 
 }  // namespace detail
+
+bool IsParquetVersionAtLeast2_0(ParquetVersion::type parquet_version) {
```

Comment:
Should we refactor this to `bool IsParquetVersionAtLeast(ParquetVersion::type parquet_version, ParquetVersion::type target_version)`? BYTE_STREAM_SPLIT was introduced in PARQUET_2_8 and will be extended in PARQUET_2_11. We need a way to check minimum version of an encoding.